### PR TITLE
Add v0.18.1 changelog

### PR DESCRIPTION
**Pull request by chrisdoherty4**

### CODE REVIEW

File: docs/content/en/docs/whatsnew/changelog.md [hunk 1]
```diff
@@ -15,12 +15,37 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
...
+### Tool Upgrade
+* Etcdadm Bootstrap Provider: `v1.0.9` to `v1.0.10`
+* Etcdadm Controller: `v1.0.14` to `v1.0.15`
+* EKS Distro (Kubernetes):
+	* `v1.25.14` to `v1.25.15`
+	* `v1.26.9` to `v1.26.10`
+	* `v1.27.6` to `v1.27.7`
+	* `v1.28.2` to `v1.28.3`
+* EKS Anywhere Packages Controller : `v0.3.12` to `v0.3.13`
+
```
Comment (vignesh-goutham) [inline docs/content/en/docs/whatsnew/changelog.md:29]:
```suggestion
* Cluster API Provider Cloudstack: `v0.4.9-rc7` to `v0.4.9-rc8`

```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 2]
```diff
@@ -702,7 +728,7 @@ ManageForeignRoutingPolicyRules=no
 
 - Support to create and manage workload clusters [#94](https://github.com/aws/eks-anywhere/issues/94)
 - Support for upgrading eks-anywhere components [#93](https://github.com/aws/eks-anywhere/issues/93), [Cluster upgrades]({{< relref "../clustermgmt/cluster-upgrades" >}})
-  - IMPORTANT: Currently upgrading existing flux manged clusters requires performing a few [additional steps]({{< relref "../clustermgmt/cluster-upgrades" >}}). The fix for upgrading the existing clusters will be published in `0.6.1` release 
+  - IMPORTANT: Currently upgrading existing flux manged clusters requires performing a few [additional steps]({{< relref "../clustermgmt/cluster-upgrades" >}}). The fix for upgrading the existing clusters will be published in `0.6.1` release
```
Comment (drewvanstone) [inline docs/content/en/docs/whatsnew/changelog.md:731]:
```suggestion
  - IMPORTANT: Currently upgrading existing flux managed clusters requires performing a few [additional steps]({{< relref "../clustermgmt/cluster-upgrades" >}}). The fix for upgrading the existing clusters will be published in `0.6.1` release
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 3]
```diff
@@ -15,12 +15,38 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
...
 {{% /alert %}}
 
+## [v0.18.1](https://github.com/aws/eks-anywhere/releases/tag/v0.18.1)
+
+### Tool Upgrade
+* EKS Distro (Kubernetes):
+	* `v1.25.14` to `v1.25.15`
+	* `v1.26.9` to `v1.26.10`
+	* `v1.27.6` to `v1.27.7`
+	* `v1.28.2` to `v1.28.3`
```
Comment (g-gaston) [inline docs/content/en/docs/whatsnew/changelog.md:25]: should this also (or instead) include the ekd-d release?
maybe a question more for the template itself than this PR

**Review by g-gaston — COMMENTED** [review]
/lgtm

File: docs/content/en/docs/whatsnew/changelog.md (hunk 3 above)
Comment (chrisdoherty4) [reply docs/content/en/docs/whatsnew/changelog.md:25]: Possibly. This was created based on the changes in build tooling. The versions are more or less copied verbatim.

**Pull request merged**